scripts/check-gomod-version.sh

```diff
@@ -3,12 +3,15 @@ set -eo pipefail
 
 [[ $2 = "-v" ]] && set -x;
 
+# go.work and `go list -modfile=...` seem to interact badly, and complain about duplicates.
+# the good news is that you can just drop that and `cd` to the folder and it works.
```

Comment:
they way they "interact badly" varies by Go version :\
_none_ of the `go.work`-supporting ones (currently) seem to like it though.  they all break or misbehave or complain.